client/rpcserver/handlers.go

```diff
@@ -617,9 +616,7 @@ func handleAppSeed(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
 	// Zero seed and hex representation after use.
 	seedHex := fmt.Sprintf("%x", seed[:])
 	defer func() {
-		for i := range seed {
-			seed[i] = 0
-		}
+		encrypt.Zero(seed)
 		seedHex = ""
```

Comment:
I'm not sure that clearing a string this way is effectual.

https://stackoverflow.com/questions/39968084/is-it-possible-to-zero-a-golang-strings-memory-safely

The method to clear string memory seems overkill. It's probably fine.

Reply:
Just removed because ineffectual.